tests/emukit/benchmarking/test_benchmark_result.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_array_equal
+
+from emukit.benchmarking.loop_benchmarking.benchmark_result import BenchmarkResult
+
+
+def test_benchmark_result_initialization():
+    br = BenchmarkResult(['loop1', 'loop2'], 5, ['mae', 'mse'])
+
+    mae_data = br.extract_metric_as_array('loop1', 'mae')
+
+    assert mae_data.size == 0
+    assert mae_data.shape == (5, 0)
+
+    mse_data = br.extract_metric_as_array('loop1', 'mse')
+
+    assert mse_data.size == 0
+    assert mse_data.shape == (5, 0)
+
+
+def test_benchmark_result_invalid_names():
```

Comment:
what about `test_benchmark_result_invalid_extract_names`? I was not sure if you were testing init names.

Reply:
Changed